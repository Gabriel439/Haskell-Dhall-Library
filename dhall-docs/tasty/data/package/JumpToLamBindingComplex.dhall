{-|
A complex example of jump-to-definition on let-bindings
-}
let fun =
    位(a : Text) ->
    位(b : Text) ->
    位(c : Text) ->
    位(d : Text) ->
    [ a ++ b, b ++ c, c ++ d, d ++ a ]

in fun
