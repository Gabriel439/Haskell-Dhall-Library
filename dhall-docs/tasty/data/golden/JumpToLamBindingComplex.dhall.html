<!DOCTYPE HTML>
<html
  ><head
    ><title
    >/JumpToLamBindingComplex.dhall</title
    ><link href="index.css" type="text/css" rel="stylesheet"
    /><link href="https://fonts.googleapis.com/css2?family=Fira+Code:wght@400;500;600;700&amp;family=Lato:ital,wght@0,400;0,700;1,400&amp;display=swap"
            type="text/css" rel="stylesheet"
    /><script src="index.js" type="text/javascript"
    /><meta charset="UTF-8"/></head
  ><body
    ><div class="nav-bar"
      ><img src="dhall-icon.svg" class="dhall-icon"
      /><p class="package-title"
      >test-package</p
      ><div class="nav-bar-content-divider"
      /><a id="switch-light-dark-mode" class="nav-option"
      >Switch Light/Dark Mode</a></div
    ><div class="main-container"
      ><h2 class="doc-title"
        ><span class="crumb-divider"
        >/</span
        ><a href="index.html"
        >test-package</a
        ><span class="crumb-divider"
        >/</span
        ><span href="index.html" class="title-crumb"
        >JumpToLamBindingComplex.dhall</span></h2
      ><a data-path="/JumpToLamBindingComplex.dhall"
          class="copy-to-clipboard"
        ><i
          ><small
          >Copy path to clipboard</small></i></a
      ><br
      /><div class="doc-contents"
        ><p
        >A complex example of jump-to-definition on let-bindings</p>
</div
      ><h3
      >Source</h3
      ><div class="source-code"
        ><pre
          >{-|<br
          />A complex example of jump-to-definition on let-bindings<br
          />-}<br
          />let <span id="var4-5" data-variable="var4-5"
                      class="variable-decl"
          >fun</span
          > =<br
          />    位(<span id="var5-7" data-variable="var5-7"
                        class="variable-decl"
          >a</span
          > : Text) -&gt;<br
          />    位(<span id="var6-7" data-variable="var6-7"
                        class="variable-decl"
          >b</span
          > : Text) -&gt;<br
          />    位(<span id="var7-7" data-variable="var7-7"
                        class="variable-decl"
          >c</span
          > : Text) -&gt;<br
          />    位(<span id="var8-7" data-variable="var8-7"
                        class="variable-decl"
          >d</span
          > : Text) -&gt;<br
          />    [ <a href="#var5-7" data-variable="var5-7"
                     class="variable-use"
          >a</a
          > ++ <a href="#var6-7" data-variable="var6-7" class="variable-use"
          >b</a
          >, <a href="#var6-7" data-variable="var6-7" class="variable-use"
          >b</a
          > ++ <a href="#var7-7" data-variable="var7-7" class="variable-use"
          >c</a
          >, <a href="#var7-7" data-variable="var7-7" class="variable-use"
          >c</a
          > ++ <a href="#var8-7" data-variable="var8-7" class="variable-use"
          >d</a
          >, <a href="#var8-7" data-variable="var8-7" class="variable-use"
          >d</a
          > ++ <a href="#var5-7" data-variable="var5-7" class="variable-use"
          >a</a
          > ]<br
          /><br
          />in <a href="#var4-5" data-variable="var4-5" class="variable-use"
          >fun</a
          ><br/></pre></div></div></body></html>